src/main/resources/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource/config-detail.jelly

```diff
@@ -35,4 +35,7 @@
     //     projectOwner.item(0).value = currentPath;
     // })
   </script>
+  <f:entry title="${%Mark unstable as succeess}" field="markUnstableAsSuccess">
```

Comment:
Property name is ambiguous:  one can think, that success result will be set in jenkins, not in GitLab, to my mind it's better to have this property in **Additional** group and **GitLab notifications** subgroup

Reply:
Since there is such an additional parameter as "Skip pipeline status notifications", it was decided to name the section as "Mark unstable build as successful on Gitlab"

Reply:
Now it looks like this
![изображение](https://user-images.githubusercontent.com/47037396/216579287-d5d55ceb-264a-4f6f-96fa-423148d9ed9e.png)